### DISCUSSION TITLE
only can use one model name (BeatRice), other Models not working

### DISCUSSION THREAD

**Discussion by navidcms2** [w-okada/voice-changer#1059] [Q&A]
Hello
I just Installed the GPU Coda for Windows
MMVCServerSIO_win_onnxgpu-cuda_v.1.5.3.17b
It took me a while and finally I could use the Model BeatRice
But when I use any other Model, there is no sound or feedback
I Downloaded some Models and add theme myself, But same result, no sound
only when i choose BeatRice Model I can Hear anything
what to do?
thanks

**Comment by PCNWN** [comment]
me too @w-okada pls help

**Reply by type4101** [reply]
Its been more than five months.  Any luck fixing the voice changer or getting a response from the developers?  None of the voices work for me, only passthrough.

**Comment by AnnoyingFox** [comment]
Could you give more information?
Computer specs?
What does the console says?
Which settings are you using?
